sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/WordElement.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * Creates an element of type {@link ElementType#WORD}
+ */
+@Immutable
+public final class WordElement extends Element {
+    /*
+     * Type of the element.
+     */
+    private final ElementType elementType;
+
+    /**
+     * Creates an {@link Element element} of type Word.
+     *
+     * @param text Text content of the extracted field.
+     * @param boundingBox Bounding box of the field value, if appropriate.
+     */
+    public WordElement(String text, BoundingBox boundingBox) {
```

Comment:
Should this have a public ctor? Do we expect users to create instances of this class?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/9619